packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyUtils.ts

```diff
@@ -158,6 +158,10 @@ const extractNested = (propertyTraversal: string[], propertyFields: Field[]): Pr
           nestedContentField.pathToPrimaryClass &&
           nestedContentField.pathToPrimaryClass.length > 1
         ) {
+          // Hardcoded navigation to external source repository metadata.
+          if (nestedContentField.contentClassInfo.name === "BisCore:RepositoryLink") {
```

Comment:
Could you add unit tests for `GroupPropertyUtils.ts`?

Reply:
In another PR.